pkg/cacheutil/memcached_client.go

```diff
@@ -244,6 +276,18 @@ func newMemcachedClient(
 	c.duration.WithLabelValues(opGetMulti)
 	c.duration.WithLabelValues(opSet)
 
+	c.valueSize = promauto.With(reg).NewHistogramVec(prometheus.HistogramOpts{
+		Name: "thanos_memcached_operation_value_size_bytes",
```

Comment:
Maybe `thanos_memcached_operation_data_size_bytes`? Give a `GetMulti()` works with multiple values, renaming `value` into `data` is more correct?